com.unity.cinemachine/Editor/Editors/CinemachineSplineSmootherEditor.cs

```diff
@@ -0,0 +1,35 @@
+using UnityEditor;
+using UnityEditor.Splines;
+using UnityEditor.UIElements;
+using UnityEngine.UIElements;
+
+namespace Unity.Cinemachine.Editor
+{
+    [CustomEditor(typeof(CinemachineSplineSmoother))]
+    class CinemachineSplineSmootherEditor : UnityEditor.Editor
+    {
+        public override VisualElement CreateInspectorGUI()
+        {
+            var ux = new VisualElement();
+            ux.Add(new HelpBox("Spline Smoother adjusts the spline's knot settings to maintain smoothness "
+            + "suitable for camera paths.  Do not adjust the tangents manually; they will be overwritten by the smoother.", 
+            HelpBoxMessageType.Info));
+            ux.Add(new PropertyField(serializedObject.FindProperty("AutoSmooth")));
```

Comment:
I wish I had known about `nameof` earlier!  Thanks for this.  Elsewhere I used some awkward expressions to avoid hardcoded strings.  I like this much better.